src/main/java/edu/kis/vh/nursery/HanoiRhymer.java

```diff
@@ -1,17 +1,28 @@
 package edu.kis.vh.nursery;
 
-public class HanoiRhymer extends defaultCountingOutRhymer {
+public class HanoiRhymer extends DefaultCountingOutRhymer {
 
-int totalRejected = 0;
+	private int totalRejected = 0;
+
+	public int getTotalRejected() {
+		return this.totalRejected;
+	}
+
+	public void setTotalRejected(int totalRejected) {
+		this.totalRejected = totalRejected;
+	}
 
```

Comment:
Encapsulation in "accessory i hermetyzacja" concerned only 'total' getter and Node class.